src/pages/NewProperty.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import CreatePropertyForm from '../components/Property/CreatePropertyForm';
+import CreateAddressForm from '../components/Property/CreateAddressForm';
+import CreateReservationCriteriaForm from '../components/Property/CreateReservationCriteriaForm';
+
+const NewProperty = () => {
+  const [propertyId, setPropertyId] = useState(null);
+  const [step, setStep] = useState(0);
+  const navigate = useNavigate();
+  // const FormTitles = ['Property', 'Address', 'Reservation Criteria'];
```

Comment:
- [ ] Kindly, remove this old piece of commented code in order to improve the repo's cleanliness. 👍